### PR TITLE
Feat : Pages(UI) and Note component

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added UI of home, landing, login and signup pages.
Added react router 
Added Note & Rich text editor components with basic functionalities

Paths for :
login -> "/login"
signup -> "/signup"
home -> "/home"
landing -> "/"

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
clean coding. Good to merge

File: src/components/Landing-Page/content.css [hunk 1]
```diff
@@ -27,4 +27,9 @@
     font-size: 1.2rem;
     background-color: #aff;
     border: 2px solid #055;
```
Comment (zoyatanweer) [inline src/components/Landing-Page/content.css]: use var for color

File: src/components/Login/login.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.login-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid black;
+    border-radius: 5px;
+    width: 20rem;
+    margin: auto;
+    background-color: #ccfab1;
```
Comment (zoyatanweer) [inline src/components/Login/login.css]: use var for color

**Review by zoyatanweer — APPROVED** [review]
- code looks clean
- use var for color wherever required.
- UI looks great.
- good to merge.

**Pull request merged**